examples/example_device/example_device.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import signal
+from datetime import datetime
+from os import listdir
+from os.path import join, isfile
+from pathlib import Path
+from random import random
+from time import sleep
+
+from astarte.device import Device
+
+_ROOT_DIR = Path(__file__).parent.absolute()
+_INTERFACES_DIR = os.path.join(_ROOT_DIR, 'interfaces')
+_DEVICE_ID = 'DEVICE_ID_HERE'
+_REALM = 'REALM_HERE'
+_CREDENTIAL_SECRET = 'CREDENTIAL_SECRET_HERE'
+_PAIRING_URL = 'https://api.astarte.EXAMPLE.COM/pairing'
+
+
+class ProgramKilled(Exception):
+    pass
+
+
+def signal_handler(signum, frame):
+    print("Shutting Down...")
+    raise ProgramKilled
+
+
+def _load_interfaces() -> [dict]:
+    files = [join(_INTERFACES_DIR, f) for f in listdir(_INTERFACES_DIR) if isfile(join(_INTERFACES_DIR, f))]
+    interfaces = []
+    for file in files:
+        with open(file, 'r') as interface_file:
+            interfaces.append(json.load(interface_file))
+    return interfaces
+
+
+def main():
+    device = Device(device_id=_DEVICE_ID, realm=_REALM, credentials_secret=_CREDENTIAL_SECRET,
+                    pairing_base_url=_PAIRING_URL, persistency_dir=str(_ROOT_DIR))
+    for interface in _load_interfaces():
+        device.add_interface(interface)
+    device.connect()
+
+    sensor_id = "b2c5a6ed-ebe4-4c5c-9d8a-6d2f114fc6e5"
+    device.send("org.astarte-platform.genericsensors.AvailableSensors", f"/{sensor_id}/name", "randomThermometer")
+    device.send("org.astarte-platform.genericsensors.AvailableSensors", f"/{sensor_id}/unit", "°C")
+
+    max_temp = 30
+    while True:
+        temp = round(random() * max_temp, 2)
+        device.send("org.astarte-platform.genericsensors.Values", f"/{sensor_id}/value", temp, datetime.now())
```

Comment:
```suggestion
        device.send("org.astarte-platform.genericsensors.Values", f"/{sensor_id}/value", temp, datetime.now(tz=timezone.utc))
```

Reply:
This removes the warning regarding the timezone that would otherwise appear when running the example.